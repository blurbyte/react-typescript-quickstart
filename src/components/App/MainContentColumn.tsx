import styled from '@styles/styledComponents';

const MainContentColumn = styled.section`
  grid-column: 1 / -2;
`;

export default MainContentColumn;
