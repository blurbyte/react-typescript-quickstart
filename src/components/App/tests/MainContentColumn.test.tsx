import * as React from 'react';
import { shallow } from 'enzyme';

import MainContentColumn from '../MainContentColumn';

describe('MainContentColumn', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<MainContentColumn />);
    expect(wrapper).toMatchSnapshot();
  });
});
