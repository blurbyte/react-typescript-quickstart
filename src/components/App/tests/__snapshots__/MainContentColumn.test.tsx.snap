// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContentColumn renders correctly 1`] = `
.c0 {
  grid-column: 1 / -2;
}

@media (max-width:62.5em) {
  .c0 {
    grid-column: 1 / -1;
  }
}

<section
  className="c0"
/>
`;
